client/src/Api.elm

```diff
@@ -102,6 +102,7 @@ type Endpoint
     | AdminNewProduct
```

Comment:
Seems like the deleted product is still shown in the product list page:

[Screencast from 08-08-2025 10:50:30 AM.webm](https://github.com/user-attachments/assets/585ab694-71f6-403c-bcbd-974996664e68)

Reply:
Should be fixed now, please check once again